crates/bevy_scene/src/scene_loader.rs

```diff
@@ -6,6 +6,10 @@ use bevy_reflect::TypeRegistryArc;
 use bevy_utils::BoxedFuture;
 use serde::de::DeserializeSeed;
 
+// Written in "one line" to insert it into the module overview.
+#[doc = "[`AssetLoader`]: <ul>\
+<li>`.scn`, `.scn.ron` -> [`DynamicScene`](`crate::dynamic_scene::DynamicScene`)</li>\
+</ul>"]
```

Comment:
![image](https://user-images.githubusercontent.com/715360/122057912-632f8780-cdeb-11eb-934a-b0b4bbfb6d67.png)
(in the `scene` module docs)

It's pretty much impossible to tell what a particular `AssetLoader` does from its signature(s), but if I'm not mistaken they can be summed up entirely with lists like this one.

I've inlined it into the module documentation here only because it's very short. I think I'd put the list behind the break if it has more than two or three entries.